server/flyway/local_sql/V1000__update_test_user_cognito_id.sql

```diff
@@ -5,10 +5,17 @@ UPDATE app_fam.fam_user
 SET cognito_user_id = 'test-idir_b5ecdb094dfb4149a6a8445a0mangled@idir'
 WHERE user_name = 'COGUSTAF';
 
+-- add user_guid for COGUSTAF, as we use it as requester for tests, and requester should have user_guid
+-- please note that this user is different from the test user we use in auth_funtion tests
 UPDATE app_fam.fam_user
-SET user_guid = 'b5ecdb094dfb4149a6a8445a0mangled'
+SET user_guid = 'B5ECDB094DFB4149A6A8445A0MANGLED'
```

Comment:
I feel maybe we switch to use all upper case for tests, cause I think from Cognito we'll get all upper case

Reply:
Thanks. Good change I think!